nodes/parachain/src/cli.rs

```diff
@@ -20,12 +20,11 @@ use crate::chain_spec;
 use std::{ops::Deref, path::PathBuf};
 use structopt::StructOpt;
 
-pub const DEFAULT_RUNTIME: &str = "peregrine";
-pub const DEFAULT_PARA_ID: &str = "2000";
+pub(crate) const DEFAULT_RUNTIME: &str = "peregrine";
 
 /// Sub-commands supported by the collator.
 #[derive(Debug, StructOpt)]
-pub enum Subcommand {
+pub(crate) enum Subcommand {
```

Comment:
with `pub(crate)` we get unused warnings. These things are never used outside anyways...